### PR TITLE
[WIP] Documentation & Notebook updates

### LINKED ISSUE #260: [BUG] Auto Accidents Example Mapping Issue
For the [auto accidents example notebook](https://docs.rapids.ai/api/cuxfilter/stable/examples/auto_accidents_example.html) when scrubbing years between 2012-2014 the map layout forms a grid or the data points become too large.  The number of data points seems correct - possibly bad data?

This is using 0.19 fresh install.

### PR DESCRIPTION
**Pull request by AjayThorve**
WIP PR to add documentation for the latest changes introduced in #324 and support for google colab and aws studio lab.

Also closes #260 by adding a note to dataset info in the readme about corrupted data

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -1,10 +1,77 @@
...
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb) 
+    [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb)
+</br>
+
+
+## Setup Remote Environments
+### Amazon Sagemaker Studio Lab
+
+[Amazon SageMaker Studio Lab](https://studiolab.sagemaker.aws/faq) is a free online web application for learning and experimenting with data science and machine learning using Jupyter notebooks and is free to use.
```
Comment (exactlyallan) [inline notebooks/README.md:31]: Amazon SageMaker Studio Lab is a free ML development environment that provides the compute, storage (up to 15GB), and security —all at no cost (currently). This includes GPU notebook instances.

File: notebooks/README.md [hunk 2]
```diff
@@ -1,10 +1,77 @@
...
+conda install ipykernel
+
+# for stable rapids version
+conda install -c rapidsai -c nvidia -c numba -c conda-forge \
+    cuxfilter python=3.7 cudatoolkit=11.5
+
+# for nightly rapids version
+conda install -c rapidsai-nightly -c nvidia -c numba -c conda-forge \
+    cuxfilter python=3.7 cudatoolkit=11.5
+```
```
Comment (exactlyallan) [inline notebooks/README.md:47]: I'd make a note that these are sample conda install snippets, but they should go to the rapids get started page for the latest (w/ link). Then we can update them less periodically.

**Review by exactlyallan — COMMENTED** [review]
Just a few minor copy fixes.

File: notebooks/README.md (hunk 1 above)
Comment (AjayThorve) [reply notebooks/README.md]: fixed

File: notebooks/README.md (hunk 2 above)
Comment (AjayThorve) [reply notebooks/README.md:47]: added a note as mentioned. Thanks :)

**Comment by AjayThorve** [comment]
@gpucibot merge

**Review by exactlyallan — APPROVED** [review]

**Comment by AjayThorve** [comment]
@gpucibot merge

**Pull request merged**